resources/warnings.json

```
@@ -16926,7 +16926,7 @@
     "versions": [
       {
         "lastVersion": "2.5",
-        "pattern": ".*"
+        "pattern": "1[.].*|2[.][012345](|[.-].*)"
```

```suggestion
        "pattern": "(1|2[.][0-5])(|[.-].+)"
```

For consistency and brevity.